style.css

```diff
@@ -440,11 +471,11 @@ filter: invert(15%);
 
 .container, .tips{
 	position: relative;
-	width: 275px;
-	height: 275px;
+	width: 314px;

```

Comment:
I have decided to make the images much bigger for use in larger screens and to fill the page